src/components/SearchError.tsx

```diff
@@ -0,0 +1,47 @@
+import {View} from 'react-native'
+
+import {atoms as a, useBreakpoints} from '#/alf'
+import * as Layout from '#/components/Layout'
+import {Text} from '#/components/Typography'
+import {Cross} from './icons/Cross'
+
+export function SearchError({
+  title,
+  children,
+}: {
+  title?: string
+  children?: React.ReactNode
+}) {
+  const {gtMobile} = useBreakpoints()
+
+  // NOTE (chenyu): asking Hannah about the new fill color for the SVG
+  return (
+    <Layout.Content>
+      <View
+        style={[
+          a.align_center,
+          a.gap_4xl,
+          {
+            paddingHorizontal: 10,
+            paddingVertical: 150,
+          },
+        ]}>
+        <Cross width={32} fill={'#D1D8E0'} />
+        <View
+          style={[
+            a.flex,
+            a.flex_col,
```

Comment:
Views actually default to `flex` and `flex-direction: column` in React Native, so you can drop these